[PATCH] fix a building failure

Upstream-Status: Inappropriate [configuration]

building failed since native does not generate import_executables.cmake
In fact, our building system will export the needed commands

Signed-off-by: Roy Li <rongqing.li@windriver.com>
---
 CMakeLists.txt | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index c1ce1c5..83b3a7e 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -350,11 +350,6 @@ MYSQL_CHECK_READLINE()
 
 CHECK_JEMALLOC()
 
-IF(CMAKE_CROSSCOMPILING)
-  SET(IMPORT_EXECUTABLES "IMPORTFILE-NOTFOUND" CACHE FILEPATH "Path to import_executables.cmake from a native build")
-  INCLUDE(${IMPORT_EXECUTABLES})
-ENDIF()
-
 #
 # Setup maintainer mode options by the end. Platform checks are
 # not run with the warning options as to not perturb fragile checks
-- 
1.9.1

